scRNA-seq/directory_structure.txt

```diff
@@ -0,0 +1,90 @@
+scRNA-seq
```

Comment:
Do we expect to add this file and `scRNA-seq/data/tabula-muris/fastq-raw` to the repo?

Reply:
`scRNA-seq/data/tabula-muris/fastq-raw` is already in the repo, but `scRNA-seq/data/tabula-muris/fastq` will be deleted in the near future (when I get to those notebooks).

When you say "this file", do you mean `scRNA-seq`? That is there as the head of the tree; I replaced `.` with that for specificity. I expect more updates to this file to keep it in sync (and not enormous) and I expect to use parts of it for figures to parallel the workflow map.

Reply:
I meant this tree - is this text file of the tree going to be in the repo? Sounds like yes because you're going to use it in figures.

And I was referring to this file: https://github.com/AlexsLemonade/training-modules/pull/249/files#diff-9d023e04dc426e6bed88e5e3ada5ff55 that add might make more sense/be obvious in the context of the snakemake changes you've already made but it's not obvious to me now.

Reply:
Oh, yeah, I do think I'll leave the tree file in the repo. Somebody might want to look back at it, and it doesn't seem to hurt.

As to `scRNA-seq/data/tabula-muris/fastq-raw`: Yes, that will stay. The tentative plan was to use one of the fastq files from that directory for the alevin quantification. And maybe have the second exercise be to repeat some of the analysis with a different starting file. These would replace the subset files in `scRNA-seq/data/tabula-muris/fastq`